js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/SensorAutomaterializationEvaluationHistoryTable.tsx

```diff
@@ -83,6 +84,8 @@ export const SensorAutomaterializationEvaluationHistoryTable = ({
     pageSize: PAGE_SIZE,
   });
 
+  useBlockTraceOnQueryResult(queryResult, 'AssetSensorTicksQuery');
```

Comment:
Might be worth driving through all the pages and putting this hook in most places where we are using `useQueryRefreshAtInterval`. I think generally the one that refreshes is the main data query for the page